xbot_service_interface_py/xbot_service_interface/shell.py

```
@@ -0,0 +1,890 @@
+        else:
+            object.__setattr__(self, name, value)
+
+    # ------------------------------------------------------------------
+    # Helper methods
+    # ------------------------------------------------------------------
+
+    def wait_connected(self, timeout: float = 10.0) -> bool:
+        """Block until the service is claimed, or timeout seconds pass."""
+        iface = object.__getattribute__(self, '_iface')
+        if iface._connected:
+            _print(f"[green]✓[/green] Already connected", plain="✓ Already connected")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)